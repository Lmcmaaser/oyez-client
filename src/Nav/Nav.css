/* mobile view */
  .Nav {
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: space-evenly;
    list-style-type: none;
    padding: 15px;
    margin-top: 10px;
    background-color: white;
    font-size: 18px;
  }
/* desktop view */
@media screen and (min-width: 600px) {
  .Nav {
    display: block;
    height: 100%;
    width: 120px;
    position: fixed;
    z-index: 1;
    top: 0;
    left: 0;
    background-color: #111;
    overflow-x: hidden;
    padding-top: 45px;
    margin: 0px;
  }

  nav a {
    padding: 6px 8px 6px 16px;
    text-decoration: none;
    font-size: 22px;
    color: #818181;
    display: block;
  }

  h4 {
    padding: 6px 8px 6px 16px;
    text-decoration: none;
    font-size: 22px;
    color: #818181;
    display: block;
  }
}
