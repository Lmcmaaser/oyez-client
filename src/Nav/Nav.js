import React from 'react';
import { Link } from 'react-router-dom';
import Content from '../content';
import './Nav.css';

export default class Nav extends React.Component {
  render () {
    return (
      <Content className='Nav'>
        <Link to='/'>
          Home
        </Link>
        <Link to='/report'>
          Report
        </Link>
        <Link to='/all'>
          View All
        </Link>
        <Link to='/state'>
          View State
        </Link>
        <Link to='/zipcode'>
          View Zip Code
        </Link>
      </Content>
    )
  }
}
