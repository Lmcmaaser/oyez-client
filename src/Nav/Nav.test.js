import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Nav from './Nav';

it('renders a .Nav by default', () => {
  const wrapper = shallow(<Nav />);
  expect(toJson(wrapper)).toMatchSnapshot();
});

it('renders without crashing', () => {
  shallow(<Nav />);
});
