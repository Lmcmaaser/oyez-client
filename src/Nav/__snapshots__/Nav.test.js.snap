// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a .Nav by default 1`] = `
<Content
  className="Nav"
>
  <div
    className="nav-contacts"
  >
    <Link
      to="/"
    >
      <GrHome />
    </Link>
  </div>
  <div
    className="nav-contacts"
  >
    <Link
      to="/report"
    >
      <GrAddCircle />
    </Link>
  </div>
  <div
    className="nav-contacts"
  >
    <Link
      to="/search"
    >
      <GrSearch />
    </Link>
  </div>
</Content>
`;
